packages/devtools_app/lib/src/screens/vm_developer/object_viewport.dart

```diff
@@ -0,0 +1,123 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:vm_service/vm_service.dart' hide Stack;
+
+import '../../primitives/auto_dispose_mixin.dart';
+import '../../primitives/history_manager.dart';
+import '../../shared/common_widgets.dart';
+import '../../shared/history_viewport.dart';
+import '../../shared/utils.dart';
+import '../debugger/program_explorer_controller.dart';
+import 'vm_class_screen.dart';
+import 'vm_class_screen_controller.dart';
+import 'vm_developer_common_widgets.dart';
+
+/// Displays the VM information for the currently selected object in the program explorer.
+class ObjectViewport extends StatefulWidget {
```

Comment:
I don't think this needs to be a `StatefulWidget` since we don't actually store any state in `_ObjectViewportState`.